### ISSUE TITLE
Issues with cellphonedb plot after fresh installation

### ISSUE THREAD

**Issue by fbnrst** [ventolab/CellphoneDB#35]
I installed cellphonedb today. Running the method worked like a charm, but getting the plots to work took me some time. I ran in the following issues:

1) The REAME suggests to install like this:
```bash
conda create -n cpdb python=3.7
pip install cellphonedb
```
This leads me directly into #19. If I install python 3.8, as specified in `environment.yml`, this does not happen.

2) I ran into an issue where the plotting command would throw an error like "graphics api version mismatch". I debugged this and found that my `rpy2` was not talking to the R in the `cpdb` environment but to some other R installation. I solved it by installing `rpy2` in the environment using conda (`conda install rpy2`) prior to installation of cellphonedb. 

In summary, I would suggest to update the README to suggest something like:
```bash
conda create -n cpdb python=3.8 rpy2
pip install cellphonedb
```
and change `environment.yml` to (adding `rpy2`)
```
name: cpdb
dependencies:
  - python=3.8
  - r-base
  - r-ggplot2
  - r-pheatmap
  - pip
  - rpy2
  - pip:
    - cellphonedb
 ```

**Comment by shanshenbing** [comment]
Hey，I used your yaml file to create a new cpdb env and I found same issue as markupsafe version error as [issue19](https://github.com/ventolab/CellphoneDB/issues/19). After downgrade markupsafe  to 2.0.1, plot fun can run correctly.
